docs/configuration/index.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 id: configuration
 title: Configuration
-sidebar_position: 2
+sidebar_position: 3
```

Comment:
This is the same position as "Usage". @madebyrogal can you update the positions everywhere, and sync the changes to the previous versions?